ios/RNCImageEditor.mm

```diff
@@ -56,6 +56,10 @@ - (void) cropImage:(NSString *)uri
   }
   NSString *displaySize = data.resizeMode();
   NSURLRequest *imageRequest = [NSURLRequest requestWithURL:[NSURL URLWithString: uri]];
+  CGFloat compressionQuality = 1;
+  if (data.quality().has_value()) {
+    compressionQuality = *data.quality();
```

Comment:
there's no validation whether the provided value is in range 0...1 - if one passes negative value or value higher than 1, it may result in unexpected behavior

Reply:
Added `quality` validation